source/daplink/interface/target_flash.c

```diff
@@ -71,6 +88,9 @@ static error_t flash_func_start(flash_func_t func)
 
     if (last_flash_func != func)
     {
+#ifdef TARGET_MCU_PSOC6
+			  // there is absolutely no point in doing uninit / init when executing the erase-program-verification sequence for PSOC6
```

Comment:
Please remove this ifdef. The flash algo specification requires init/uninit to be called. If you can come up with a generic mechanism to avoid this, then it will be accepted.

Reply:
I came up with a general mechanism to avoid this. Please consider the approach with algo_flags in the program_target structure.

Reply:
code has been redesigned to use g_target_family-> apsel